src/tools/pontoon/ui.md

```diff
@@ -122,3 +150,55 @@ The `TERMS` tab shows the definition and translation of a term, in case the sour
 The `COMMENTS` tab is used to display existing **source string comments**, or add new ones. Source string comments, unlike translation comments, are associated with the string: it’s possible to have a comment in this section even if the string doesn’t have any suggestion or translation yet.
 
 They’re designed for team members to have a conversation about the source string, for example to clarify its meaning, or to get more information from project managers.
+
+### Machinery
+
+Machinery available from the translation view shows possible translations from a variety of sources. These sources include:
+* [Pontoon’s internal translation memory](translate.md#downloading-and-uploading-translations).
+* [Microsoft Terminology](https://www.microsoft.com/Language/).
+* [Google Translate](https://translate.google.com).
+* [SYSTRAN](https://www.systran.net/).
+* [Caighdean](https://github.com/kscanne/caighdean).
+* [Bing Translator](https://www.bing.com/translator) (not currently enabled on pontoon.mozilla.org).
+
+In addition, the user has the ability to search for translations containing words via [`Concordance search`](#concordance-search).
+
+In the tab, the number of entries is visible alongside the `MACHINERY` title in white. If any of the machinery matches are from translation memory, the number of matches will appear separately in green. For example, the screenshot below shows `2+1`, where the green `2` represents the two matches from translation memory and the `1` represents a machinery entry from Google Translate.
+
+![Machinery tab](../../assets/images/pontoon/ui/machinery.png)
+
+At the top of each entry, a diff view compares the current source string and the source string from the machinery entry. Strikethrough text highlighted in red indicates text that is present in the source string but not in the machinery source string, while text highlighted in green indicates text available only in the machinery source string.
+
+To the right of the entry, the number in green shows the percent match between the machinery string and the source string. The higher the percentage, the more likely the machinery is to be useful and a 100% match indicates that the source and machinery string are the same. The machinery source is listed in gray. Upon clicking the gray text, the machinery source will be opened in a new window. The green superscript to the right indicates the number of matches for the entry in the translation memory source.
```

Comment:
```suggestion
To the right of the entry, the number in green shows the percent match between the machinery source string and the current source string. The higher the percentage, the more likely machinery is to be useful; a 100% match indicates that the sources for the current string and for the machinery string are identical.

The origin of the machinery entry is listed in gray above the source string. Clicking the gray text will open the origin in a new window. The green superscript to the right indicates the number of matches for the entry in the translation memory.
```

I just realized this paragraph doesn't make a lot of sense, also mixing `source` for different meaning.

Reply:
Much simpler, thanks for the suggestion.